src/pages/VaccinePage.js

```diff
@@ -1,125 +1,196 @@
 /* eslint-disable react/forbid-prop-types */
-/**
- * mSupply Mobile
- * Sustainable Solutions (NZ) Ltd. 2016
- */
-
 import React from 'react';
-import { connect } from 'react-redux';
-import { StyleSheet, Text, FlatList } from 'react-native';
+import { FlatList, StyleSheet } from 'react-native';
+import moment from 'moment';
 import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import * as Animatable from 'react-native-animatable';
+import {
```

Comment:
This is just a copy paste of `NewVaccineModulePage` and then I deleted `VaccinePage` - there are a couple things to fix in this component where we have made issues, but I have no fixed anything in here. OK maybe a prop type or two, I think